test/test_cli.rb

```diff
@@ -1,441 +1,357 @@
 # frozen_string_literal: true
+
 require_relative 'helper'
 require 'sidekiq/cli'
+require 'sidekiq/launcher'
 require 'tempfile'
 
-class Sidekiq::CLI
```

Comment:
I guess it's ugly to patch the class for testing purposes this way. But I really liked the technique [here](https://github.com/mperham/sidekiq/blob/master/lib/sidekiq/launcher.rb#L63):
```
private unless $TESTING
```
^_^